model/src/w3odatmd.F90

```diff
@@ -1,5 +1,20 @@
+!> @file w3odatmd.F90
+!> @ brief Define data structures to set up wave model grids and aliases
+!>         to use individual grids transparently.
```

Comment:
In general, it's not worth trying to maintain any special spacing with doxygen comments. Doxygen formats everything nicely, ignoring your whitespace, and it's just extra work for the programmers who have to maintain the code.

So our documentation blocks on NCEPLIBS look like this:

```
!> @file
!> @brief Create new GRIB2 file with fields from an existing GRIB2 file.
!> @author Stephen Gilbert @date 2004-05-17

!> Program reads selected GRIB2 fields from a file, adds a TOC Flag
!> Field separator block and WMO Header in front of each GRIB2 field,
!> and writes them out to a new file. The output file is in the format
!> required for TOC's FTP Input Service, which can be used to
!> disseminate the GRIB2 bulletins.
!>
!> Passing filenames into this program is done in a special
!> way. Before running this program, set the following environment
!> variables:
!>
!> Variable | Meaning
!> ---------|--------
!> FORT11 | Name of the input GRIB2 file.
!> FORT31 | Name of the input GRIB2 index file.
!> FORT51 | Name to use for the output GRIB2 file.
!>
!> @note The "EXTRACT" variable in the namelist allows users to choose
!> whether they want the entire GRIB2 message containing the requested
!> field (extract=.false.), or a GRIB2 message containing only the
!> requested field (extract=.true.). Both options return the same
!> message if the requested field is the only field in the GRIB2
!> message.
!>
!> ### Program History Log
!> Date | Programmer | Comments
!> -----|------------|---------
!> 2004-05-17 | Gilbert | Initial
!> 2012-06-14 | Vuong | Modified write statement with format specifier
!> 2012-10-22 | Vuong | Changed variable envvar to character*6
!> 2016-10-15 | Vuong | Increased length of file name to 200 Characters

```

(Note that this uses markdown to add a table to the docs. But in all cases, just one space is used as whitespace.)

Reply:
That's great you can include markdown tables like that.  I did discover doxygen will ignore your spacing attempts, this was accidentally left in.  I'll change it.